WcaOnRails/app/models/user.rb

```diff
@@ -68,6 +68,13 @@ def name_must_match_person_name
     end
   end
 
+  validate :dob_must_be_on_the_past
+  def dob_must_be_on_the_past
+    if dob.year >= Date.today.year
+      errors.add(:dob, "year of birth must be on the past")
```

Comment:
I think it's "in the past", but I'm not a native speaker, so take this with a grain of salt.